compose.yaml

```
@@ -0,0 +1,27 @@
+services:
+  self-service:
```

```suggestion
  app:
```
Acha que faz sentido man?

Pra não ficar `self-service-self-service`?